alibuild_helpers/build_template.sh

```diff
@@ -12,6 +12,25 @@ set +h
 function hash() { true; }
 export WORK_DIR="${WORK_DIR_OVERRIDE:-%(workDir)s}"
 
+# Insert our own git and curl wrapper scripts into $PATH, patched to use the
+# system OpenSSL, instead of the one we build ourselves.
+tmpdir=$(mktemp -d)
```

Comment:
I would do it somewhere in the $WORKDIR.

Reply:
You're right, that's much better.